src/components/SimplePopup/index.test.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import jest from 'jest-mock';
```

Comment:
remove

Reply:
when I use the normal 
```
import jest from 'jest';
```
then I run `jest.fn()` in my test, it states, that `fn()` is undefined.. which is why I use `jest-mock`

Reply:
no need to import jest... just use the eslint rule as stated above

Reply:
refactored